FlowCrypt/Controllers/Inbox/InboxViewController+TableView.swift

```diff
@@ -402,6 +402,9 @@ extension InboxViewController {
         hideSpinner()
 
         guard let indexToUpdate = getUpdatedIndex(for: inboxItem) else {
+            // Just reload data when index was not found in rare cases.
+            // https://github.com/FlowCrypt/flowcrypt-ios/issues/2366
+            tableNode.reloadData()
```

Comment:
Oops, I meant not just table reloading (it probably won't change anything, as it'll be the same data already used by table), but performing [refresh from remote mail server](https://github.com/FlowCrypt/flowcrypt-ios/blob/5fc8ba868fb755c02670b96463d64673780cbff3/FlowCrypt/Controllers/Inbox/InboxViewController.swift#L344)

Reply:
Yep! You are right. Just fixed it